src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -204,7 +204,7 @@ public sealed class IidChangePointEstimator : TrivialEstimator<IidChangePointDet
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        /// [!code-csharp[MF](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Timeseries.cs?range=6-12,17-37,231-293
```

Comment:
>c/Timeseries.cs?range=6-12,17-37,231-293 [](start = 88, length = 40)

please leave the ranges in the TImeSeries files. The samples on the time series are all in one file, that's why we still need to reference them by region.  Issue #1577 has an ask to split them up; than we can reference them by file name only.